client/src/components/Tickets/TicketModal.scss

```diff
@@ -102,6 +102,9 @@
   }
   .link {
     color: var(--global-color-accent--normal);
+    display: inline-flex;
+    position: relative;
+    float: right;
```

Comment:
**Discrepancy.** Missing border in the middle of the rows in the table ([annotated screenshot](https://user-images.githubusercontent.com/62723358/140996545-f0ed5f54-1ddf-4bd4-a8b6-e99b521cb9ea.png)).

```suggestion
    text-align: right;
```

<details>
<summary>Screenshots</summary>

| Before | After |
| - | - |
| ![Before Alignment Fix](https://user-images.githubusercontent.com/62723358/140996165-740703aa-13de-485b-b0ff-65ededa93d73.png) | ![After Alignment Fix](https://user-images.githubusercontent.com/62723358/140996166-36045ed7-e56b-4747-8c90-8761c0317c1a.png) |

[After Alignment Fix (with code change)](https://user-images.githubusercontent.com/62723358/140996169-69caa152-4d4d-4590-872d-e1be88aabb34.png)

</details>